jSka/config/properties.py

```diff
@@ -0,0 +1,18 @@
+import os
+from jSka.core.exceptions import ImproperlyConfigured
+
+def get_env_variable(var_name):
+
+    try:
+        return os.environ[var_name]
+    except:
+        message = "Set the {} environment variable".format(var_name)
+        raise ImproperlyConfigured(message)
+
+
+INGEST_DIR=get_env_variable('JSKA_INGEST_DIR')
```

Comment:
For testing / development it might be useful to have defaults for these so you don't need to set up an environment.